packages/shared-ux/prompt/no_data_views/src/data_view_illustration.tsx

```diff
@@ -8,545 +8,22 @@
 
 import React from 'react';
 import { useEuiTheme } from '@elastic/eui';
-import { css } from '@emotion/css';
+import { css as emotion } from '@emotion/css';
+
+import svg from './data_view_illustration.svg';
 
 export const DataViewIllustration = () => {
   const { euiTheme } = useEuiTheme();
   const { colors } = euiTheme;
 
-  const dataViewIllustrationVerticalStripes = css`
-    fill: ${colors.fullShade};
-  `;
-
-  const dataViewIllustrationDots = css`
-    fill: ${colors.lightShade};
+  const css = emotion`
+    .dataViewIllustrationVerticalStripes {
+      fill: ${colors.fullShade};
+    };
+    .dataViewIllustrationDots {
+      fill: ${colors.lightShade};
+    }
```

Comment:
@clintandrewhall Unfortunately this breaks dark mode. These styles can't apply to images pulled into `src` as they're just static url references. We have 2 options:

1. Find a way to optimize the original SVG (the textures are what causes this too be so large)
1. Export 2 versions as `.png`s (a light version and dark) at 2x the size 

![image](https://user-images.githubusercontent.com/549577/172421474-48c43d60-7748-404f-b64b-f6d06e83f951.png)

Reply:
You can only style SVG's with CSS if the SVG is **inline**. Source url's have absolutely now knowledge of CSS. The updates just reference the static `.svg` as a url:
<img width="453" alt="Screen Shot 2022-06-07 at 11 56 30 AM" src="https://user-images.githubusercontent.com/549577/172426723-42ea0503-544c-4b34-89c4-4be7348735df.png">

Reply:
Yep, I got confused... happens sometimes, I guess.  The good news is I was able to reduce the SVG bloat at least a bit by moving the SVG out of the React component and into a static file-- it removes all of the `React.createComponent` calls in the compiled code.

Follow-up here: https://github.com/elastic/kibana/pull/133821